src/CognitiveServices/CognitiveServices/CognitiveServicesAccount/NewAzureCognitiveServicesAccountDeployment.cs

```diff
@@ -61,6 +61,15 @@ public class NewAzureCognitiveServicesAccountDeploymentCommand : CognitiveServic
             Mandatory = true,
             ParameterSetName = DefaultParameterSet,
             ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Cognitive Services Deployment Sku.")]
+        [ValidateNotNullOrEmpty]
+        public Sku Sku { get; set; }
```

Comment:
Looks like you added a required parameter without any default value. This is a breaking change and not allowed in this sprint release. Please either give the Sku property a default value which can make the existing cmdlet work for our customers. Otherwise, you have to wait for the next breaking change release that is around Ignite 2023.

Reply:
I added this by mistake, it is not a required, I fixed this in the new commit. Thank you.